app/models/restaurant.rb

```diff
@@ -0,0 +1,22 @@
+
+class NameValidator < ActiveModel::Validator
```

Comment:
Not a mandatory thing, but we usually respect each class with it's own file.

Reply:
done. moved to a different file called restaurant_name_validation